cardano-config/src/Cardano/TracingOrphanInstances/Network.hs

```diff
@@ -82,6 +86,10 @@ showPoint verb pt =
     NormalVerbosity -> take 7
     MaximalVerbosity -> id
 
+-- | Hex-encode and render a 'HeaderHash' as text.
+renderHeaderHash :: ConvertRawHash blk => proxy blk -> HeaderHash blk -> Text
+renderHeaderHash p = Text.decodeLatin1 . B16.encode . (toRawHash p)
```

Comment:
I suspect the parentheses are redundant?

Reply:
Oh, yeah that's true.